viz-lib/src/visualizations/chart/Editor/GeneralSettings.jsx

```diff
@@ -107,6 +107,15 @@ export default function GeneralSettings({ options, data, onOptionsChange }) {
     }
   }
 
+  function handleAxesInversion() {
+    // moves any item in the right Y axis to the left one
+    const seriesOptions = mapValues(options.seriesOptions, series => ({
+      ...series,
+      yAxis: 0,
```

Comment:
By doing this, every time horizontal chart changes state, all series go back to left Y axis.